src/platform/testing/e2e/mockapi.js

```diff
@@ -47,10 +47,20 @@ function makeMockApiRouter(opts) {
 
     mockResponses[token] = mockResponses[token] || {};
     mockResponses[token][verb] = mockResponses[token][verb] || {};
-    mockResponses[token][verb][path] = {
-      status: req.body.status,
-      value: req.body.value,
-    };
+    if (req.body.query) {
```

Comment:
The reasons for the changes in here:

Normally, the mock api stores responses by path and verb, so that if you make a request to the same url, you get the same data back. It ignores the query params. This is problematic for us, because we have an endpoint that returns two different types of responses depending on the query params (type being cc or va). So, we need to have mocks that are different depending on the query params. But, it's also problematic to include the query param in the path that the mock api considers, since those params might be dynamic (like start dates).

So, my solution was to allow you to pass a `query` property that the mock api would use to do a regex test agains the query string. So we don't have to specify the whole query string, just the piece we care about.